package.json

```diff
@@ -9,7 +9,8 @@
     "url"
   ],
   "scripts": {
-    "test": "jest --coverage && eslint . && check-dts && size-limit",
+    "test:short": "c8 --check-coverage uvu . .test.js$",
```

Comment:
Let’s move `c8` to `test` to make `test:short` (or `unit` for better consistency with other projects) faster.